### ISSUE TITLE
[Feature Request] Batch Renormalization

### ISSUE THREAD

**Issue by arthurflor23** [keras-team/keras#19857]
Hi,

The `renorm` parameter in the`BatchNormalization` layer is available in Keras 2 ([link](https://github.com/keras-team/keras/blob/601488fd4c1468ae7872e132e0f1c9843df54182/keras/layers/normalization/batch_normalization.py#L98C7-L98C13)), but not in Keras 3.
Is there any intention to migrate this option to Keras 3 as well? It's very good for training with mini-batches.

In any case, thank you in advance.

**Comment by sachinprasadhs** [comment]
`renorm` flag was removed in `Keras 3` to make `Keras` compatible across all backend and we don't have any plan to add it in `Keras 3`.

For more context refer: https://github.com/keras-team/keras/issues/19063#issuecomment-1915518601
